cp1404practicals/prac_5/emails.py

```diff
@@ -0,0 +1,25 @@
+
+emails_dict = {}
+email_lengths = []
+name_lengths = []
+new_email = str(input("Email: "))
+
+while new_email != "":
+    email_split_by_dots = new_email.split(".")
+    email_further_split_by_ats = email_split_by_dots[1].split("@")
+    name = email_split_by_dots[0].title() + " " + email_further_split_by_ats[0].title()
```

Comment:
I think the task wanted this a separate function.
"Use a separate function to get the name from the email as in the example below."
Otherwise looks good

Reply:
Good catch. Fixed

Reply:
changes look good